lib/core/api.js

```diff
@@ -217,23 +212,13 @@ function postBlockToPlatform(data, groupId, callback) {
       callback(jqxhr.responseJSON, 0);
     }
   };
-  if(!_.isEmpty(groupId)){
-    api.log('postBlockToPlatform: for other user');
-    tidepool.uploadDeviceDataForUser(data, groupId, function(err, result) {
-      if (err) {
-        return sad(err, err);
-      }
-      return happy(result);
-    });
-  } else {
-    api.log('postBlockToPlatform: as logged in user');
-    tidepool.uploadDeviceDataForUser(data, function(err, result) {
-      if (err) {
-        return sad(err, err);
-      }
-      return happy(result);
-    });
-  }
+
+  tidepool.uploadDeviceDataForUser(data, groupId, function(err, result) {
```

Comment:
Do you want to put an api.log back here? Maybe including logging of the groupId?